crates/ruff_workspace/src/options.rs

```diff
@@ -2266,6 +2266,31 @@ impl Pep8NamingOptions {
 #[serde(deny_unknown_fields, rename_all = "kebab-case")]
 #[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
 pub struct PycodestyleOptions {
+    /// The maximum [width](http://www.unicode.org/reports/tr11/#Overview) to allow for [`line-too-long`] violations. By default,
+    /// this is set to the value of the global `line-width` option.
+    ///
+    /// Overriding the global [`line-width`] allows to use a lower limit for Ruff's formatter and warn about
+    /// extra-long lines that can't be split by the formatter using [`line-too-long`]:
+    ///
+    /// ```toml
+    /// line-width = 88 # The formatter breaks line's with a width of 88.
```

Comment:
Nit: two spaces between content and `#`

Reply:
Doesn't this just apply to Python code?

Reply:
Hmm, I'm not sure what I would do in TOML, actually, but you're probably right.

Reply:
I certainly wouldn't do this in TOML. I'm not even used to do it in Python 😆